azure_arc_servers_jumpstart/gcp/terraform/scripts/vars.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Replace with values for your environment surrounded by ""
```

Comment:
Change to 
# <--- Change the following environment variables according to your Azure Service Principle name --->